src/dask_awkward/layers/layers.py

```diff
@@ -107,20 +107,30 @@ def mock_empty(self, backend: BackendT = "cpu") -> AwkwardArray:
         )
 
 
+def io_func_rer_wrapped(func: ImplementsIOFunction) -> bool:
+    return hasattr(func, "__rer_wrapped__")
```

Comment:
It looks like this is inspired by `__wrapped__`. For our naming, perhaps `_raise_on_error_wrapped` might be better?